### PR TITLE
gallery page

### PR DESCRIPTION
**Pull request by yussifweb**
gallery page by Team HotShots

### CODE REVIEW

File: src/assets/scss/app.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // CORE
 @import "bootstrap/scss/bootstrap-grid.scss";
+@import "bootstrap/scss/bootstrap.scss";
```
Comment (kingflamez) [inline src/assets/scss/app.scss:3]: We are only using the grid system for bootstrap. Other CSS should be created by you. Please remove it

File: src/assets/scss/app.scss [hunk 2]
```diff
@@ -16,3 +17,4 @@
 //PAGES
 @import "./pages/home";
 @import "./pages/faq";
+@import "./pages/hotshots";
```
Comment (kingflamez) [inline src/assets/scss/app.scss:20]: This is meant to be /pages/gallery

File: src/pages/hotshots.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+              src={img.src}
+              text={img.text}
+            />
+          );
+        })}
+      </div>
+    </div>
+    </DefaultLayout>
+  );
+}
```
Comment (kingflamez) [inline src/pages/hotshots.jsx:37]: Please the page should be `gallery` not hotshots

**Review by kingflamez — CHANGES_REQUESTED** [review]
Please review

**Comment by yussifweb** [comment]
fixing all errors

**Pull request closed without merging**